App/HejDTO/Sources/HejDTO/_Common/Common.Self.Link.swift

```diff
@@ -0,0 +1,13 @@
+
+import Foundation
+
+extension Common {
+    
+    public struct SelfLink: Codable, Equatable {
```

Comment:
Could you check in what other places you can reuse this type? :)

Reply:
Of course, I will fix it tomorrow:)